spec/imsc-hrm.html

```diff
@@ -244,6 +244,8 @@ <h2>Terms and Definitions</h2>
       A sequence of consecutive <a>intermediate synchronic documents</a> conforms to the Hypothetical Render Model if
       is processed without error as defined in <a href="#hypothetical-render-model-general"></a>.
     </p>
+    <p class="note">The results of processing a <a>Document Instance</a> that is not an <a>IMSC Document Instance</a> are
+      undefined.</p>
```

Comment:
This new text may not be as true or as helpful as it seems: the results are _defined_ but they may not be useful or meaningful. Trying to think of a better way of saying this... How about:

```suggestion
Although it may be possible to process a <a>Document Instance</a> that is not an <a>IMSC Document Instance</a>,
the results should not be taken to reflect that Document Instance's complexity, since other features may be used that increase complexity in ways not considered by the Hypothetical Render Model.
```

```

Reply:
I think this is saying too much: the bottom line is that the HRM is defined only for IMSC documents and, hence, applying the HRM to other kinds of documents is undefined, e.g. processing might fail in strange and unexpected ways.

Reply:
My reading of the spec is that you will always get _some_ answer: I don't think I could construct an example that would fail entirely, but I'm fairly sure it would be possible to construct an example that would pass in a misleading way by using features that are simply not counted in the HRM. Happy to be proven wrong!

Reply:
Ok. What about: _Applying the HRM to a document that does not conform to IMSC yields results that do not reflect the complexity of the document._

Reply:
> Applying the HRM to a document that does not conform to IMSC yields results that do not reflect the complexity of the document.

s/do/may (or "might") => that works for me

Reply:
_might not_ implies a significant probability that the results do reflect the complexity. I would think it is the opposite: the results are unlikely to reflect the complexity of the document. What about _that do not reflect_ -> _that are unlikely to reflect_?

Reply:
Hmm, there are many reasons why a document could not conform to IMSC, some of which probably don't really affect complexity, others of which probably affect complexity greatly. We can't sensibly speculate on this unknown.

I think we should leave this open rather than implying some probability in one direction or another.

I don't read _might not_ as implying any probability, but if you think others will, I'm happy to use a different term. On the other hand, _are unlikely to_ does imply a probability one way, so I'm less happy with that.

What about "may"?

Reply:
See revised text.